usr/share/rear/conf/default.conf

```diff
@@ -2847,6 +2847,15 @@ because the disks and filesystems of the target system are mounted there.
 # This command is shown to the user and added to the predefined bash history
 # in the ReaR recovery system to make the manual backup restore easier:
 REQUESTRESTORE_COMMAND=
+#
+# Set the finished and abort files to enable non-interactive REQUESTRESTORE mode.
+# ReaR will wait for the finished file to appear and continue with the recovery,
+# or abort the recovery if the abort file appears. The content of the abort file is
+# shown and logged as abort reason.
+# Instead of the manual prompt ReaR will simply show an info message and the disk
+# usage of the target filesystems.
```

Comment:
I would add here that if these variables are empty, `REQUESTRESTORE` works as before.